gwpy/signal/filter_design.py

```diff
@@ -306,7 +309,51 @@ def bilinear_zpk(zeros, poles, gain, fs=1.0, unit='Hz'):
     return dzeros, dpoles, dgain
 
 
-def parse_filter(args, analog=False, sample_rate=None):
+def convert_to_digital(filter, sample_rate, unit='Hz'):
+    """Convert an analog filter to digital via bilinear functions.
+
+    This function always returns in zpk format.
+
+    Parameters
+    ----------
+    filter: `tuple`
+        filter values
+
+    sample_rate: `float`
+
+    unit: `str`
+
+    Returns
+    -------
+    dform : `str`
+        type of filter, 'zpk' format
+
+    dfilter: 'tuple'
+        digital filter values
+    """
+
+    # This will always end up returning zpk form.
+    # If FIR, bilinear will convert it to IIR.
+    # If IIR, only if p_i = -2 * fs will it yield poles at zero.
+    # See gwpy/signal/tests/test_filter_design for more information.
+
+    form, filter = parse_filter(filter)
+
+    if form not in ('ba', 'zpk'):
+        raise ValueError(f"Cannot convert {form}, only 'zpk' or 'ba'")
+
+    if form == 'ba':
+        b, a = filter
+        if len(a) < len(b) and a[0] == 1:
+            a = numpy.append(a, numpy.zeros(len(b)-len(a)))
+        filter = signal.tf2zpk(b, a)
+
+    filter = bilinear_zpk(*filter, fs=sample_rate, unit=unit)
+
+    return parse_filter(filter)
```

Comment:
second parse_filter could be replaced with return "zpk", filter

Reply:
Given that `filter` is always going to be the output of `bilinear_zpk`, we know the form, so shouldn't have to parse it. Can we just return the output of `bilinear_zpk`?